Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -2628,3 +2628,50 @@ function Remove-CustomRole {
     }
 }
 
+<#
+    .Synopsis
+    This function allows the customer to restart HA on a cluster. This might be needed to clear HA alarms.
+    .EXAMPLE
+    Once the function is imported: Restart-ClusterHA -cluster_name "Cluster-1"
+#>
+function Restart-ClusterHA {
+    [AVSAttribute(30, UpdatesSDDC = $false)]
+    param(
+        [Parameter(Mandatory = $true,
+            HelpMessage = "What cluster to restart HA on.")]
+        [String]
+        $cluster_name
+    )
+
+    # Get the cluster object
+    try {
+        $cluster = Get-Cluster -Name $cluster_name -ErrorAction Stop
+    }
+    catch {
+        Write-Error "Cluster $cluster_name not found."
```

Comment:
Is this an error that could be ignored, i.e does it make sense for the script to continue? If not then use `throw` instead, so that this error is surfaced as the reason the command failed.

Reply:
same logic in subsequent cases...